code/app/src/main/java/com/adobe/marketing/edge/identity/app/ui/CustomIdentityFragment.kt

```diff
@@ -37,10 +42,17 @@ class CustomIdentityFragment : Fragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
+        val LOG_TAG = "Custom_Identity_Fragment"
```

Comment:
This is a variable not a constant, you can use the same approach as per my other comment about constants.

Reply:
Usage updated!